test/util/test_tools.hpp

```diff
@@ -95,6 +98,40 @@ size_t count_ops_of_type(std::shared_ptr<ngraph::Function> f)
     return count;
 }
 
+template <typename T>
+void init_int_tv(std::shared_ptr<ngraph::runtime::Tensor> tv,
```

Comment:
pass Tensor*
Same comment for init_real_tv

Reply:
Done.